python-regression/util/transaction_bundle_logic/bundle_logic.py

```diff
@@ -0,0 +1,22 @@
+from iota import BundleHash, Fragment
+from iota.crypto import HASH_LENGTH
+from iota.crypto.kerl import Kerl
+
+
+def finalize(bundle):
```

Comment:
just remind me why you didn't use PYOTA's finalize

Reply:
If we use the PYOTA finalize it modifies the obsolete tag value and leads to unpredictable milestone indexes